Hunting Queries/AWSS3/AWSBucketAPILogs-SuspiciousDataAccessToS3BucketsfromUnknownIP.yaml

```diff
@@ -22,7 +22,7 @@ query: |
   AwsBucketAPILogs_CL 
   | where EventTime TimeGenerated between(starttime..endtime)
```

Comment:
Hi @referefref, 
On what column you are applying filter EventTime or TimeGenerated. Can you please check and confirm.
Thanks

Reply:
Hi,
This line of the query was not modified at all from the original.
Thanks.